<?xml version="1.0" ?>
<sdf version="1.5">

    <model name="tree2">
      <static>true</static>
      <link name="tree">
	<visual name="body_visual">
	    <pose>0 0 0 0 0 0</pose>
	    <geometry>
	      <mesh>
		<scale>0.9 0.9 1.0</scale>
		<uri>model://tree2/meshes/tree02.stl</uri>
	      </mesh>
	    </geometry>
	    <material>
	      <script>
		<name>Gazebo/Green</name>
		<uri>file://media/materials/scripts/gazebo.material</uri>
	      </script>
	    </material>
	  </visual>
      </link>
    </model>

</sdf>
